java-cfenv-boot-pivotal-scs/src/main/java/io/pivotal/cfenv/boot/scs/config/CfSpringCloudConfigClientProcessor.java

```diff
@@ -13,7 +13,7 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-package io.pivotal.cfenv.boot.scs;
+package io.pivotal.cfenv.boot.scs.config;
```

Comment:
While moving to a config package is nicer, it isn't the kind of change to make in a point release (which I am thinking where we will add this functionality).

Reply:
Ok, that makes sense. I'll move it all into one big happy package! :)